AI_ML/REGRESSION/LEAST_SQUARES/LEAST_SQUARES.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import traceback
+from flojoy import flojoy, DataContainer
+
+
+@flojoy
+def LEAST_SQUARES(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
```

Comment:
I think it would make sense for this node to also support passing a single `ordered_pair`, do you agree @jackparmer @smahmed776?